app/routes.js

```diff
@@ -137,12 +152,7 @@ export const routes = (authProvider) => {
       });
     }
 
-    const context = await getSupplierSearchPageErrorContext({
-      orderId,
-      validationErrors: response.errors,
-    });
-
-    return res.render('pages/sections/supplier/search/template.njk', addContext({ context, user: req.user, csrfToken: req.csrfToken() }));
+    return res.redirect(`${config.baseUrl}/organisation/${orderId}/supplier/search`);
```

Comment:
where is the redirect to the 0 found page?

Reply:
the throw new ErrorContext on like 146